source/com.microsoft.tfs.client.clc/src/com/microsoft/tfs/client/clc/vc/commands/CommandResolve.java

```diff
@@ -203,7 +203,9 @@ public void run() throws ArgumentException, MalformedURLException, CLCException,
          */
         boolean containsSymlink = false;
         for (final Conflict conflict : conflicts) {
-            if (FileSystemUtils.getInstance().getAttributes(conflict.getTargetLocalItem()).isSymbolicLink()) {
+            if (conflict.getTargetLocalItem() != null
+                && conflict.getTargetLocalItem().length() > 0
```

Comment:
I agree, but I would hesitate to change the code since I don't know the best way to test it.